lib/manipulation.rb

```diff
@@ -0,0 +1,14 @@
+class Parser
+    def initialize(file)
+        @file = file
+    end
+    
+    def openfile
+        unless File.exist?(@file)
+            puts "Arquivo não encontrado"
```

Comment:
Só lembrando que puts retorna `nil`, apesar de jogar a msg pro `STDOUT`. Se quiser retornar o valor da string, pode trocar por `p`.

```suggestion
            p "Arquivo não encontrado"
```